src/extensions/registries/index.ts

```diff
@@ -7,3 +7,4 @@ export * from "./status-bar-registry"
 export * from "./kube-object-detail-registry";
 export * from "./kube-object-menu-registry";
 export * from "./cluster-feature-registry"
+export * from "./resource-status-registry"
```

Comment:
This should be named similar to other `kube-object-*` registries, maybe `kube-object-status-registry`?

Reply:
Renamed